integration-tests/pom.xml

```diff
@@ -104,6 +104,7 @@
         <module>bootstrap-config</module>
         <module>injectmock</module>
         <module>reactive-messaging-amqp</module>
+        <module>resteasy-client</module>
     </modules>
```

Comment:
Also the module needs to be added here: https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml#L392 (and please add 10 minutes to the timeout a few lines above)